### ISSUE TITLE
Implement msc2545 Custom Stickers

### ISSUE THREAD

**Issue by ata-star** [vector-im/element-android#6193]
#### What would you like to do?
make and use msc2545 stickers
#### Why would you like to do it?
because its fun and makes element compete with Telegram
#### How would you like to achieve it?
with this Issue

didnt find an issue regarding this

### Have you considered any alternatives?

the default stickers are boring

the homeserver admin dictates the integrateons

fluffychat is the only option for mobile and its ok but misses minor stuff i also want

### Additional context

no, not really

**Comment by bahuma20** [comment]
Probably duplicate of https://github.com/vector-im/element-android/issues/6193

**Comment by ata-star** [comment]
what?

**Comment by bahuma20** [comment]
Oh sorry, i meant this is probably a duplicate of https://github.com/vector-im/element-android/issues/8060

**Comment by ata-star** [comment]
nah, its newer. This here is the original

**Comment by lukasfri** [comment]
Any news?

**Comment by htayj** [comment]
This would be a great feature to have. Since most other clients have it, it looks a bit weird on matrix when people on other clients respond to posts with the custom emoji.

**Comment by Nostramadeus** [comment]
Big agree everyone, +1 on this awesome feature.

**Comment by jhebert8** [comment]
It should be standard, I agree...

**Comment by maxwell-kalin** [comment]
3 years already?
hey everyone! 

Element went corpo-mode

**Comment by Doc-4** [comment]
How has this not been implemented? I remember the initial implementation into matrix years ago, it's baffling the largest client still doesn't support it after so many years. This is the only thing stopping me from using the client everywhere. Even if you don't fully implement the feature, at least implement seeing them when used inline in text, empty messages might as well be bugs!

**Comment by maxwell-kalin** [comment]
> How has this not been implemented? I remember the initial implementation into matrix years ago, it's baffling the largest client still doesn't support it after so many years. This is the only thing stopping me from using the client everywhere. Even if you don't fully implement the feature, at least implement seeing them when used inline in text, empty messages might as well be bugs!

Yeah, i think they found out that element can make money and stopped makeing the client fun and make it a corporate-use only
